src/storage/admin/AdminProcessor.h

```diff
@@ -58,7 +58,6 @@ class TransLeaderProcessor : public BaseProcessor<cpp2::AdminExecResp> {
         if (partMeta.hosts_.size() == 1) {
             LOG(INFO) << "Skip transfer leader of space " << spaceId
                       << ", part " << partId << " because of single replica.";
-            this->pushResultCode(cpp2::ErrorCode::E_TRANSFER_LEADER_FAILED, partId);
```

Comment:
Why remove the error msg? The user should know if the BALANCE operation worked. And if the operation failed, we should provide a clear cause like "To balance the xx, the graph spaces must have more than one replicas."

Reply:
This is way more complicated than you thought. Actually, there won't be this phase for 1 replica balance

Reply:
the single replica may not require transfer leader so it's fine @randomJoe211